doc/nrf/ug_matter_configuring_protocol.rst

```diff
@@ -0,0 +1,75 @@
+.. _ug_matter_configuring_protocol:
+
+Configuring Matter in |NCS|
+###########################
+
+.. contents::
+   :local:
+   :depth: 2
+
+This page describes what is needed to start working with Matter in the |NCS|.
+
+.. _ug_matter_configuring_requirements:
+
+Required components for Matter network
+**************************************
+
+The Matter network requires the following components to properly operate:
+
+* Matter controller - configured either on PC or mobile
+* Thread Border Router - required by the Matter controller
+
+For information about how to configure these components, read :ref:`ug_matter_configuring`.
+
+.. _ug_matter_configuring_mandatory:
+
+Mandatory configuration
+***********************
+
+To use the Matter protocol, enable the :kconfig:`CONFIG_CHIP` Kconfig option.
+Setting this option enables the Matter protocol stack and other associated Kconfig options, including :kconfig:`CONFIG_CHIP_ENABLE_DNSSD_SRP` that is required for the discovery of the Matter device using DNS-SD.
+
+After that, you have to set the :kconfig:`CONFIG_CHIP_PROJECT_CONFIG` Kconfig options and define the path to the configuration file that specifies Vendor ID, Product ID, and other project-specific Matter settings.
+
+For instructions about how to set Kconfig options, see :ref:`configure_application`.
+
+.. _ug_matter_configuring_optional:
+
+Optional configuration
+**********************
+
+After enabling the Matter protocol and defining the path to the Matter configuration file, you can enable additional options in Kconfig.
+
+OpenThread configuration
+========================
+
+Enabling :kconfig:`CONFIG_CHIP` automatically enables the following options related to OpenThread:
+
+* :kconfig:`CONFIG_OPENTHREAD_ECDSA` and :kconfig:`OPENTHREAD_SRP_CLIENT` - enabled through :kconfig:`CHIP_ENABLE_DNSSD_SRP`
+* :kconfig:`CONFIG_OPENTHREAD_DNS_CLIENT` - enabled through :kconfig:`CONFIG_CHIP_ENABLE_DNS_CLIENT`
+
+Additionally, you can enable the support for Thread :ref:`Sleepy End Device <thread_ot_device_types>` in Matter by using the :kconfig:`CONFIG_CHIP_ENABLE_SLEEPY_END_DEVICE_SUPPORT` Kconfig option.
+This option sets :kconfig:`CONFIG_OPENTHREAD_MTD_SED` and :kconfig:`CONFIG_OPENTHREAD_MTD` Kconfig options.
```

Comment:
```suggestion
This option sets :kconfig:`CONFIG_OPENTHREAD_MTD_SED` Kconfig option.
```
`CONFIG_OPENTHREAD_MTD` is not implied, it's required for the CONFIG_CHIP_ENABLE_SLEEPY_END_DEVICE_SUPPORT option to be available.